docs/apis/plugintypes/mod_data/fields.md

```diff
@@ -1,81 +1,113 @@
 ---
-title: Database fields
-tags:
-  - mod_data
-  - datafield
-  - plugintype
-  - subplugin
-documentationDraft: true
----
 
-The [Database activity](https://docs.moodle.org/en/Database_module) included with Moodle includes support for several predefined [field types](./fields.md), including text, date, and URL. It is also possible to create new field types. For example, you might like to create:
+# Database Fields for Moodle 4.4.3
+
+*This documentation is a work-in-progress. Feel free to contribute.*
+
+The **Database activity** in Moodle allows users to create structured collections of data. It supports various predefined field types like **text**, **date**, and **URL**. Developers can extend Moodle by creating custom field types, which are beneficial for specialized uses like discipline-specific, institution-specific, or module-specific needs.
```

Comment:
I don't think any of these things need to be bolded. THe individual types _could_ use backticks.

```suggestion
The **Database activity** in Moodle allows users to create structured collections of data. It supports various predefined field types like `text`, `date`, and `URL`. Developers can extend Moodle by creating custom field types, which are beneficial for specialized uses like discipline-specific, institution-specific, or module-specific needs.
```